Gulpfile.js

```diff
@@ -253,8 +254,32 @@ gulp.task('images', ['clean'], function () {
         .pipe(gulp.dest('lib'));
 });
 
+gulp.task('greenkeeper', function () {
+    var packageJsonFilePath = path.join(__dirname, './package.json');
+    var notIgnoredDepsRe    = /testcafe|babel/;
 
-gulp.task('build', ['lint', 'server-scripts', 'client-scripts', 'styles', 'images', 'templates']);
+    return promisify(fs.readFile, Promise)(packageJsonFilePath, 'utf-8')
```

Comment:
Refactor promisified `readFile` to the var and move it tot the top of the file. BTW, `utf8` is default encoding, no need to specify it explicitly.